openpype/hosts/nuke/api/plugin.py

```diff
@@ -799,8 +800,11 @@ def __init__(self,
         self.get_file_info()
 
         self.log.info("File info was set...")
-
         self.file = self.fhead + self.name + ".{}".format(self.ext)
+        if self.ext !="mov":
+            frame = get_frame()
```

Comment:
It would be good to use this only if a padding is not already used in `self.in_path` since the variable is inheriting value from publishing context of `self.instance.data.get("path", None)` and the hash can already be part for the path, so we can use it. Read the above comment with `get_hashed_frame_padding_from_anatomy`.

In case of video file is used for rendering path then we can get the value from Anatomy. And the anatomy would be better to pass from publishing context too. It means we could have optional input arguments for `get_hashed_frame_padding_from_anatomy` to accept Anatomy.